graylog2-server/src/main/java/org/graylog/plugins/views/search/permissions/UserStreams.java

```diff
@@ -40,8 +40,22 @@ public UserStreams(StreamPermissions streamPermissions, PermittedStreams permitt
         this.permittedStreams = permittedStreams;
     }
 
-    public ImmutableSet<String> loadAll() {
-        return permittedStreams.load(streamPermissions);
+    public ImmutableSet<String> loadAllMessageStreams() {
+        return permittedStreams.loadAllMessageStreams(streamPermissions);
+    }
+
+    public ImmutableSet<String> loadAllStreams() {
+        return permittedStreams.loadAll(streamPermissions);
+    }
+
+    public ImmutableSet<String> loadStreamsForEmptyStreamsReplacement() {
```

Comment:
```suggestion
    public ImmutableSet<String> loadMessageStreamsWithFallback() {
```
How about naming it like this instead?